File: PKMania/PM-DAL/Data/Entities/Player.cs

namespace PM_DAL.Data.Entities
{
    public class Player
    {
        public int Id { get; set; }
        public string Pseudo { get; set; } = string.Empty;
        public int TableNr { get; set; }
        public int Stack { get; set; }
        public int BonusTime { get; set; }
        public DateTime EliminatedAt { get; set; }
        public Boolean Disconnected { get; set; }
    }
}

File: PKMania/PM-FrontEnd/src/app/features/tools/tournaments/models/player.ts
import { Card } from "./card";

export interface Player {
  id: number;
  pseudo: string;
  generalRanking: number;
  tableRanking: number;
  eliminated: boolean;
  sittingAtTable: number;
  seatNr: number;
  stack: number;
  moneyInPot: number;
  disconnected: boolean;
  turnToPlay: boolean;
  privateCards: Card[];
  bonusTime: number;
}
